libcodechecker/log/option_parser.py

```diff
@@ -107,7 +107,14 @@
 
 IGNORED_OPTION_MAP = {
     '-MT': 1,
+    '-MQ': 1,
     '-MF': 1,
+    '-MJ': 1,
```

Comment:
This map is the `option_parser` which filters the arguments needed from a build command to an analyzer command. It was lacking some options to ignore, which resulted in a lot of `Unhandled argument` output if `analyze` was ran with `--verbose debug`. This removes arguments that are not needed or should not be passed or should be passed in a different way to Clang.

The other, in `analysis_manager.py` uses the **original** build command verbatim, which is **NOT** translated by this parser, and thus, must be cleaned separately. This is, because the dependency listing invokes the same compiler that was used, and thus should get as much arguments as possible. But a `-MF` dependency output file specified in the build command would derail the generation to `STDOUT`.